flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -1542,7 +1543,7 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
                     this._labels[5],
                     this._lines[5],
                 ],
-                scratchpad: this.scratchpad._displayUnit._scratchpadElement.textContent,
+                scratchpad: `{${this.scratchpad._displayUnit._scratchpadElement.className}}${this.scratchpad._displayUnit._scratchpadElement.textContent}{end}`,
```

Comment:
One thing I'd note is it would be better to add a public interface to retrieve the scratchpad content/format rather than accessing private variables.

Reply:
Done